examples/IBFE/explicit/ex4/input2d

```diff
@@ -4,14 +4,14 @@ RHO = 1.0
 L   = 1.0
 
 // grid spacing parameters
-MAX_LEVELS = 1                                      // maximum number of levels in locally refined grid
-REF_RATIO  = 2                                      // refinement ratio between levels
-N = 64                                              // actual    number of grid cells on coarsest grid level
-NFINEST = (REF_RATIO^(MAX_LEVELS - 1))*N            // effective number of grid cells on finest   grid level
-DX0 = L/N                                           // mesh width on coarsest grid level
-DX  = L/NFINEST                                     // mesh width on finest   grid level
-MFAC = 2.0                                          // ratio of Lagrangian mesh width to Cartesian mesh width
-ELEM_TYPE = "TRI6"                                  // type of element to use for structure discretization
+MAX_LEVELS = 3                                     // maximum number of levels in locally refined grid
```

Comment:
Let's not change the "default" input files just yet --- instead add in a new file, `input2d.partitioner`?

Reply:
I only changed this for testing purposes (e.g., printing workloads). I'll get rid of the changes once that is done.